2021-05-21 14:56:25,499 - INFO - root - Plugin 'Shutdown' added
2021-05-21 14:56:25,504 - INFO - root - Plugin 'Backup' added
2021-05-21 14:56:25,528 - INFO - root - Plugin 'Search' added
2021-05-21 14:56:25,553 - INFO - root - Plugin 'Ico' added
2021-05-21 14:56:25,553 - INFO - root - Plugin 'Update' added
2021-05-21 14:56:25,560 - INFO - root - Plugin 'Manual' added
2021-05-21 14:56:25,560 - INFO - root - Plugin 'Repeat' added
2021-05-21 14:56:25,564 - INFO - root - Plugin 'Team' added
2021-05-21 14:56:25,568 - INFO - root - Plugin 'Social' added
2021-05-21 14:56:25,569 - INFO - root - Plugin 'Developer' added
2021-05-21 14:56:25,577 - WARNING - root - File 'alert.py' can't be loaded as a plugin: module 'opencryptobot.plugins.alert' has no attribute 'Alert'
2021-05-21 14:56:25,589 - INFO - root - Plugin 'Market' added
2021-05-21 14:56:25,595 - INFO - root - Plugin 'Compare' added
2021-05-21 14:56:25,598 - INFO - root - Plugin 'Top' added
2021-05-21 14:56:25,598 - INFO - root - Plugin 'Wallets' added
2021-05-21 14:56:25,618 - INFO - root - Plugin 'Start' added
2021-05-21 14:56:25,624 - INFO - root - Plugin 'Admin' added
2021-05-21 14:56:25,638 - INFO - root - Plugin 'Restart' added
2021-05-21 14:56:25,647 - INFO - root - Plugin 'Exchanges' added
2021-05-21 14:56:25,648 - INFO - root - Plugin 'Donate' added
2021-05-21 14:56:25,648 - INFO - root - Plugin 'Marketcap' added
2021-05-21 14:56:25,649 - INFO - root - Plugin 'People' added
2021-05-21 14:56:25,650 - INFO - root - Plugin 'Change' added
2021-05-21 14:56:25,653 - INFO - root - Plugin 'Info' added
2021-05-21 14:56:25,653 - INFO - root - Plugin 'Fiftyone' added
2021-05-21 14:56:25,655 - INFO - root - Plugin 'Help' added
2021-05-21 14:56:25,655 - INFO - root - Plugin 'Value' added
2021-05-21 14:56:25,655 - INFO - root - Plugin 'News' added
2021-05-21 14:56:25,655 - INFO - root - Plugin 'Feedback' added
2021-05-21 14:56:25,660 - INFO - root - Plugin 'Decentralized' added
2021-05-21 14:56:25,661 - INFO - root - Plugin 'Pools' added
2021-05-21 14:56:25,662 - INFO - root - Plugin 'Events' added
2021-05-21 14:56:25,663 - INFO - root - Plugin 'About' added
2021-05-21 14:56:25,668 - INFO - root - Plugin 'Roi' added
2021-05-21 14:56:25,669 - INFO - root - Plugin 'Best' added
2021-05-21 14:56:25,669 - INFO - root - Plugin 'Logfile' added
2021-05-21 14:56:25,676 - INFO - root - Plugin 'Worst' added
2021-05-21 14:56:25,676 - INFO - root - Plugin 'Bpmn' added
2021-05-21 14:56:25,676 - INFO - root - Plugin 'Description' added
2021-05-21 14:56:25,677 - INFO - root - Plugin 'Volume' added
2021-05-21 14:56:25,677 - INFO - root - Plugin 'Stats' added
2021-05-21 14:56:25,679 - INFO - root - Plugin 'Alltimehigh' added
2021-05-21 14:56:25,702 - INFO - root - Plugin 'Price' added
2021-05-21 14:56:25,949 - INFO - root - Plugin 'Whitepaper' added
2021-05-21 14:56:26,162 - INFO - root - Plugin 'Twitter' added
2021-05-21 14:56:29,442 - INFO - root - Plugin 'Global' added
2021-05-21 14:56:29,445 - INFO - root - Plugin 'Chart' added
2021-05-21 14:56:29,449 - INFO - root - Plugin 'Trends' added
2021-05-21 14:56:29,573 - INFO - root - Plugin 'Candlestick' added
2021-05-21 14:56:29,578 - INFO - root - Starting Caching
2021-05-21 14:56:31,669 - ERROR - JobQueue - An uncaught error was raised while executing job refresh
Traceback (most recent call last):
  File "/Users/ahmetcnd/.local/share/virtualenvs/openCryptoBot-D8P4_yx3/lib/python3.7/site-packages/telegram/ext/jobqueue.py", line 245, in tick
    job.run(self.bot)
  File "/Users/ahmetcnd/.local/share/virtualenvs/openCryptoBot-D8P4_yx3/lib/python3.7/site-packages/telegram/ext/jobqueue.py", line 372, in run
    self.callback(bot, self)
  File "/Users/ahmetcnd/Documents/telegrambot/openCryptoBot/opencryptobot/api/apicache.py", line 23, in refresh
    APICache.refresh_coinmarketcap_coin_list()
  File "/Users/ahmetcnd/Documents/telegrambot/openCryptoBot/opencryptobot/api/apicache.py", line 39, in refresh_coinmarketcap_coin_list
    APICache.cmc_coin_list = Market().listings()["data"]
TypeError: 'JSONDecodeError' object is not subscriptable
2021-05-21 14:56:32,029 - ERROR - root - HTTPError('404 Client Error: Not Found for url: https://api.github.com/repos/None/None/releases/latest')
2021-05-21 15:19:31,195 - ERROR - root - Timed out - {'update_id': 406258963, 'message': {'message_id': 491, 'date': 1621599561, 'chat': {'id': 1062954233, 'type': 'private', 'username': 'paemmisyon', 'first_name': 'Paemisyon'}, 'text': '/admin', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 6}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 1062954233, 'first_name': 'Paemisyon', 'is_bot': False, 'username': 'paemmisyon', 'language_code': 'en'}}, '_effective_user': {'id': 1062954233, 'first_name': 'Paemisyon', 'is_bot': False, 'username': 'paemmisyon', 'language_code': 'en'}}
